components/up-job.libsonnet

```diff
@@ -146,7 +150,63 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
     ),
   },
 
+  withLogsFile:: {
+    local job = k.batch.v1.job,
+    local container = job.mixin.spec.template.spec.containersType,
+    local u = self,
+    config+:: {
+      bashImage: error 'must provide image for bash',
```

Comment:
Is there a reason no to reuse the `observatorium/up` image for the init container? This image hash `/bin/sh`, as it's built on alpine: https://github.com/observatorium/up/blob/master/Dockerfile#L1. I think this would be a nice simplification and reduce the number of images we need to mentally consider

Reply:
The other option would be to use a configmap with the logs-file as a key. It's shifting the complexity into another place but I think it's the more correct way to load these static files

Reply:
First of all moving this away is not a good idea, because the logs file is a construct consumed only by observatorium/up. Furthermore, the need for a special image is due to the fact that the Loki API demands timestamps as unix epoch in nanoseconds (See [/loki/api/v1/push](https://github.com/grafana/loki/blob/master/docs/api.md#post-lokiapiv1push)). The most simple date implementation supporting this is GNU's coreutils' date. Alpine does not provide support for timetamps as unix epoch in nanoseconds.

Reply:
ok, not work bikeshedding this too much :)

Reply:
ok, to simplify our lives I would say to simply append some zeros to the `date` output, e.g.: `echo $(date +%s)000000000)` to get a fake nanosecond timestamp, but it's not worth bikeshedding this too much :)